freebsd/Makefile

```diff
@@ -0,0 +1,62 @@
+VALID_CATEGORIES+= igports
+UID_FILES:=${PORTSDIR}/igports/UIDs
+GID_FILES:=${PORTSDIR}/igports/GIDs
+PORTNAME=	grafsy
+PORTVERSION=	_PV_
+PORTREVISION=	1
+CATEGORIES=	igports
+
+MAINTAINER=	kajetan.staszkiewicz@innogames.com
+COMMENT=	Graphite client/proxy which listens locally on server and buffers data
+
+BUILD_DEPENDS=	go>=1.5.0:lang/go
+
+CONFLICTS_INSTALL= ig.graphite-client-*
+
+ONLY_FOR_ARCHS=	i386 amd64
+
+USE_GITHUB=	nodefault
+GH_TUPLE=	innogames:grafsy:_GC_:grafsy/src/github.com/innogames/grafsy
+GH_TUPLE+=	BurntSushi:toml:b26d9c3:toml/src/github.com/BurntSushi/toml
+GH_TUPLE+=	innogames:go-acl:c1b74e0:goacl/src/github.com/innogames/go-acl
+GH_TUPLE+=	pkg:errors:ba968bf:pkg/src/github.com/pkg/errors
+
+GRAFSY_USER=	grafsy
+GRAFSY_GROUP=	grafsy
+
+USERS=		${GRAFSY_USER}
+GROUPS=		${GRAFSY_GROUP}
+
+GRAFSY_ETCDIR=	${PREFIX}/etc/grafsy
+GRAFSY_LOGDIR=	/var/log/grafsy
```

Comment:
This is configured by the user and doesn't have defaults now. See https://github.com/innogames/grafsy/pull/2/files#diff-f72262028f884afc75798bcff6917704R1, https://github.com/innogames/grafsy/pull/2/files#diff-f72262028f884afc75798bcff6917704R1 and https://github.com/innogames/grafsy/blob/master/grafsy.toml#L15

Reply:
Question is who would make this directory. It is configured here so that when package is installed, directory will be made.

Reply:
It's puppet for us, it's something else to anybody else. If somebody use the example config - the log dir will be created by grafsy in /tmp. It doesn't make any sense to provide config, because user always have to change it for receiver server.

Another point - the lack of config should be mentioned during installation. https://github.com/innogames/grafsy/pull/2/files#diff-cd2e00c3e1393b01818e73ba9e514ad0R24

Reply:
It worth to mention that we, for example, don't provide logrotate as well. If a user has to take care about config, let him think about directory too